README.md

```diff
@@ -1,6 +1,8 @@
 Anexia Monitoring
 =================
 
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/anexia-it/anexia-monitoring-django/Module%20tests)
```

Comment:
Can you please update this to:
```
[![build-and-test actions status](https://github.com/anexia-it/anexia-monitoring-django/workflows/build-and-test/badge.svg)](https://github.com/anexia-it/anexia-monitoring-django/actions)
```
I think the bagde is generated on render as this is added to S3: https://github.com/CultureHQ/github-actions-badge

[![build-and-test actions status](https://github.com/anexia-it/anexia-monitoring-django/workflows/build-and-test/badge.svg)](https://github.com/anexia-it/anexia-monitoring-django/actions)

Reply:
![Auswahl_055](https://user-images.githubusercontent.com/55870697/94930101-38eca280-04c6-11eb-8d62-dd2961476638.png)

Den Link bekam ich direkt von shield.io